targets/XPlatCppSdk/source/licenses/jsoncpp-vc140-static-32_64.txt

```diff
@@ -0,0 +1,60 @@
+https://github.com/open-source-parsers/jsoncpp/blob/master/LICENSE
```

Comment:
Again, this is a file that we should not add unless we make a specific decision to change the way we're building from SdkGenerator.